src/main/java/org/gaul/modernizer_maven_plugin/ModernizerMojo.java

```diff
@@ -42,32 +42,33 @@
     @Parameter(defaultValue = "${project}", readonly = true)
     private MavenProject project;
 
-    @Parameter(defaultValue = "${project.build.sourceDirectory}")
+    @Parameter(property = "project.build.sourceDirectory")
```

Comment:
@hgschmie Sorry can you explain the differences between these?  I don't understand reading the javadoc:

https://maven.apache.org/plugin-tools/maven-plugin-annotations/apidocs/org/apache/maven/plugins/annotations/Parameter.html